src/main/java/bio/terra/cli/serialization/userfacing/input/UpdateReferencedBqDatasetParams.java

```diff
@@ -0,0 +1,111 @@
+package bio.terra.cli.serialization.userfacing.input;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import javax.annotation.Nullable;
+
+/**
+ * Parameters for updating a BigQuery dataset workspace referenced resource. This class is not
+ * currently user-facing, but could be exposed as a command input format in the future.
+ */
+@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "@class")
+@JsonDeserialize(builder = UpdateResourceParams.Builder.class)
+public class UpdateReferencedBqDatasetParams {
+  private final UpdateResourceParams resourceParams;
```

Comment:
nit: The other params classes have `public final` fields instead of getters, which makes the class a little less verbose. It would be nice to be consistent in this class. Ditto for the other `Update*Params` classes below.